_data/coreteam_members.yml

```diff
@@ -63,3 +63,7 @@
     a Computer Engineering undergraduate and tech enthusiast. Loves to explore the realms of software engineering and music.
     <em>More information about him here: </em> <a href="http://harishv7.github.io">harishv7.github.io</a>.
     
+
+- name: Varun
+  description: |
+    a Year 2 CS undergraduate interested in Programming Languages and Algorithms. He frequents hackathons and keeps busy with his startup. He is a varsity ultimate frisbee player and cycles around Singapore on weekends.
```

Comment:
Please hard wrap your line here. "|" allows you to do that without breaking the syntax.